CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+# Contributing to Extensible Storage Engine
+
+For us to be able to accept code contributions, they have to build and pass tests. For portions of the code that don't have a working build and working tests, we will not be able to accept contributions.
+
+Initially, none of our code builds or includes tests. Until that changes, our code synchronization process will be one-way, and we will be unable to take *any* code contributions.
+
+We will be accepting coce contributions in the future. If you are interested in contributing code, we would like to hear from you!
```

Comment:
When I added the "we would like to hear from you", I didn't give them a way to do that.  any thoughts????  I don't see a forum or chat associated with the project.

Reply:
I can clarify: interested contributors should create issues describing the issue and state their interest in the issue. If it's an existing issue that they want to work on, then comment on the issue. Something like that. I think we can manage everything using issues here on GitHub.